src/profile/utils.js

```diff
@@ -224,6 +238,10 @@ function createCardComponent({ className, tagName, innerText, parent }) {
     component.innerText = innerText;
   }
 
+  if (innerHTML) {
+    component.innerHTML = innerHTML;
```

Comment:
Are we potentially overriding the `innerText` with `innerHTML`?

Reply:
Yes. Is that a problem? Because currently, we are rendering the key-value pair as a single text. E.g: `First name: Vinit`
I have refactored it and wrapped key and value in span elements and conditionally applied red and green background classes for the diff key-value pairs.

Is this okay or can it be improved with a better solution?

Reply:
Refactored these changes. Please check it. @ankushdharkar